DEV-9/src/EntryPoint.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Two strings are given [expected the from args[]]
+ * Replace a consecutive group of characters, selected randomly in one line, by a random group of
+ * consecutive characters selected from the other.
+ * Output the original strings and the strings after processing to the console.
+ */
+public class EntryPoint {
+  final static String NO_INPUT_MESSAGE = "No input arguments. Input arguments must be passed as "+
+      "args[0] and args[1]. Program exited.";
```

Comment:
"args[0] and args[1]" doesn't make any sense for the user - it's just implementation detail